types/wlr_seat.c

```diff
@@ -146,11 +196,18 @@ struct wlr_seat_handle *wlr_seat_handle_for_client(struct wlr_seat *wlr_seat,
 
 void wlr_seat_set_capabilities(struct wlr_seat *wlr_seat,
 		uint32_t capabilities) {
-	// TODO: If e.g. pointer was removed, destroy all client pointer resources
```

Comment:
The wayland spec says we simply update the capabilities. The client must then destroy the resource. Simply destroying a resource on our side is usually not a good idea.